src/pm-conversations/PmConversationList.js

```diff
@@ -25,45 +27,48 @@ type Props = $ReadOnly<{|
 /**
  * A list describing all PM conversations.
  * */
-export default class PmConversationList extends PureComponent<Props> {
-  handleUserNarrow = (user: UserOrBot) => {
-    this.props.dispatch(doNarrow(pm1to1NarrowFromUser(user)));
+export default function PmConversationList(props: Props) {
+  const handleUserNarrow = (user: UserOrBot) => {
+    props.dispatch(doNarrow(pm1to1NarrowFromUser(user)));
   };
 
-  handleGroupNarrow = (users: PmKeyUsers) => {
-    this.props.dispatch(doNarrow(pmNarrowFromUsers(users)));
+  const handleGroupNarrow = (users: PmKeyUsers) => {
+    props.dispatch(doNarrow(pmNarrowFromUsers(users)));
   };
 
-  render() {
-    const { conversations } = this.props;
+  const { conversations } = props;
+  const mutedUsers = useSelector(getMutedUsers);
 
-    return (
-      <FlatList
-        style={styles.list}
-        initialNumToRender={20}
-        data={conversations}
-        keyExtractor={item => item.key}
-        renderItem={({ item }) => {
-          const users = item.keyRecipients;
-          if (users.length === 1) {
+  return (
+    <FlatList
+      style={styles.list}
+      initialNumToRender={20}
+      data={conversations}
+      keyExtractor={item => item.key}
+      renderItem={({ item }) => {
+        const users = item.keyRecipients;
+        if (users.length === 1) {
+          if (mutedUsers.has(users[0].user_id)) {
```

Comment:
Let's factor out this `users[0].user_id`, since it's being used twice -- that helps make it real clear it's the same user ID both times.

Also the `users[0]` only makes sense because we've just entered a condition `users.length === 1` -- basically it's some manual pattern-matching -- and so it's good to keep the `users[0]` reference right up close next to that condition. (If for example this block were to use other parts of the user object, I'd want to say `const user = users[0];` right after the `if`, and go from there.)

Reply:
Done.